### DISCUSSION TITLE
Pedigree support?

### DISCUSSION THREAD

**Discussion by schlenk** [DependencyTrack/dependency-track#1934] [Q&A] [answered]
Hi,

does DependencyTrack have CycloneDX pedigree support?

I want to get notified by DependencyTrack when one of my components has a vulnerability in one of the ancestor components of my component. 

For example, i have a Apache httpd Server with some custom patches. 

The Apache httpd component alone with cpe `cpe:2.3:a:apache:http_server:2.4.54:*:*:*:*:*:*:*` is detected and i get notifications.
If i now include this component as `<ancestors>` element in `<pedigree>`, should i get notifications for my derived component? 
And could dependency track detect `<patches>` elements with type security and detect if it matches the flagged CVE for example?

Right now i simply "lie" in my sbom, assigning the upstream cpe value to my derived component to get the notifications, but thats obviously not the ideal state.

**Comment by stevespringett** [comment, marked as answer]
Not yet. See #919
